src/main/resources/application.yml

```diff
@@ -0,0 +1,19 @@
+spring:
+  cloud:
+    gateway:
+      routes:
+      - id: catalog_route
+        uri: http://localhost:8081
+        predicates:
+        - Path=/api/**
+        filters:
```

Comment:
You can move Retry filter to **default-filters**, since it is something we would like to use independently of the route